service/user/service.go

```diff
@@ -0,0 +1,120 @@
+package user
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/stasBigunenko/monorepa/pkg/storage/newStorage"
+	"unicode"
+
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+)
+
+type UsrService struct {
+	Us      model.UserHTTP
+	storage newStorage.NewStore
+}
+
+func NewUsrService(s newStorage.NewStore) *UsrService {
+	return &UsrService{
+		storage: s,
+	}
+}
+
+func (u *UsrService) Get(_ context.Context, id uuid.UUID) (model.UserHTTP, error) {
+
+	res, err := u.storage.Get(context.Background(), id)
```

Comment:
Can pass param context as storage value

Reply:
i think better to send id value directly. Context.Value data should inform us and not control input value for the function.

Reply:
Change it to control full work line.

Reply:
changed